src/locales/it/texts.json

```diff
@@ -0,0 +1,287 @@
+    "forbidden": "Le credenziali specificate per questa richiesta non sono valide, ad esempio un token API scaduto o non valido. Fornisci altre credenziali e riprova.",
+    "invalidJsonObject": "La risposta non è un oggetto JSON valido.",
+    "invalidStacCollections": "La risposta dell'API non è un elenco valido di Collezioni STAC.",
+    "invalidStacItems": "La risposta dell'API non è un elenco valido di Elementi STAC.",
+    "labels": {
+      "cause": "Causa:",
+      "code": "Codice d'errore:",
+      "reason": "Motivo:",
+      "requestURL": "URL della richiesta:"
+    },
+    "loadApiCollectionsFailed": "Non è stato possibile caricare le Collezioni dell'API.",
+    "loadApiItemsFailed": "Non è stato possibile caricare gli Elementi dell'API.",
+    "loadFilteredItems": "Il caricamento di un elenco filtrato di Elementi non è riuscito.",
+    "loadItems": "Il caricamento dell'elenco di Elementi STAC non è riuscito.",
+    "networkError": "Questo problema può verificarsi quando i server non consentono l'accesso esterno tramite i browser web (ad esempio, quando non sono presenti le intestazioni CORS). @:errors.contactProvider",
+    "noExternalAccess": "L'accesso a cataloghi esterni non è consentito!",
+    "notFound": "La risorsa richiesta non esiste. @:errors.contactProvider",
+    "serverError": "Il server ha riscontrato un problema. @:errors.contactProvider",
+    "unauthorized": "La richiesta necessita di credenziali, ad esempio un token API. Inserisci i tuoi dati di accesso e riprova."
+  },
+  "featureExperimental": "Questa funzione è ancora sperimentale e potrebbe dare risultati inaspettati!",
+  "fullscreen": {
+    "exit": "Esci dalla modalità a schermo intero",
+    "show": "Mostra in modalità a schermo intero"
+  },
+  "goToCollection": {
+    "description": "Vai alla Collezione associata",
+    "descriptionWithTitle": "@:goToCollection.description: {title}",
+    "label": "Collezione"
+  },
+  "goToParent": {
+    "description": "Sali di un livello",
+    "descriptionWithTitle": "@:goToParent.description: {title}",
+    "label": "Su"
+  },
+  "in": "in {catalog}",
+  "index": {
+    "api": "API",
+    "catalog": "Catalogo",
+    "load": "Carica",
+    "selectStacIndex": "... o selezionane uno dall'indice STAC:",
+    "specifyCatalog": "Inserisci l'indirizzo di un Catalogo o di un'API STAC...",
+    "urlInvalid": "L'URL non è valido.",
+    "urlMissingHost": "Host mancante.",
+    "urlMissingProtocol": "Protocollo mancante."
+  },
+  "items": {
+    "filter": "Filtro",
+    "noTime": "Nessuna data/ora indicata",
+    "noneAvailableForCollection": "Non ci sono Elementi disponibili per questa Collezione."
+  },
+  "leaflet": {
+    "close": "Chiudi",
+    "noFeatureProperties": "Non ci sono altri dati disponibili per questo Elemento.",
+    "stayLayer": {
+      "error": "Non è stato possibile aggiungere i dati alla mappa"
+    },
+    "zoom": {
+      "in": {
+        "description": "Zoom in",
```

Comment:
"Zoom in / out" are the same in Italian?

Reply:
Could be. I'll change it